TrkReco/src/TrkRecoMcUtils_tool.cc

```diff
@@ -64,7 +64,7 @@ namespace mu2e {
       //                    const art::InputTag& Tag) override;
 
       virtual const SimParticle* getSimParticle(const art::Event* Event, int HitIndex) override;
-      virtual const XYZVectorF* getMom(const art::Event* Event, int HitIndex) override;
+      virtual const XYZVectorF*  getMom        (const art::Event* Event, int HitIndex) override;
```

Comment:
please remove spurious whitespace

Reply:
those are not spurious - the text alignment makes the code better readable